src/DynamoCoreWpf/ViewModels/GuidedTour/SurveyPopupViewModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dynamo.Wpf.UI.GuidedTour;
+
+namespace Dynamo.Wpf.ViewModels.GuidedTour
+{
+    public class SurveyPopupViewModel
+    {
+        #region Private Properties
+        private Survey step;
+        #endregion
+
+        /// <summary>
+        /// This property hold a reference to the Step that was created (can be Welcome, Survey, Tooltip, ExitTour). 
+        /// </summary>
+        public Survey Step
+        {
+            get
+            {
+                return step;
+            }
+            set
+            {
+                step = value;
+            }
+        }
+
+        /// <summary>
+        /// For creating a SurveyPopupViewModel we need to pass the step to be used 
+        /// </summary>
+        /// <param name="popupType">Step is a abstract class then the parameter can be any of the child clases like Welcome, Tooltip, Survey....</param>
```

Comment:
By `Step` do you mean `Survey`? `an abstract`. Also parameter is not a class, I think you mean of any Child class as type

Reply:
The comment was updated in the next commit:
https://github.com/DynamoDS/Dynamo/pull/11834/commits/84586419aa1ca8d940fcc13e6faff4ab4eba1968